### DISCUSSION TITLE
Badges no longer working

### DISCUSSION THREAD

**Discussion by cbrake** [XAMPPRocky/tokei#929] [Q&A] [answered]
The badges are no longer working:

![image](https://user-images.githubusercontent.com/402813/179826673-085d755d-9627-4cd0-9d60-1dcbbe4ad86e.png)

Does anyone know why?

**Comment by XAMPPRocky** [comment, marked as answer]
Thank you for question. I’ll fix it later today. It’s an issue with the infra being overloaded.

**Reply by cbrake** [reply]
Thanks - it's very useful -- a lot of other people must think so too!

**Reply by svenstaro** [reply]
I believe this is happening again.
